src/sentry/plugins/sentry_mail/models.py

```diff
@@ -123,6 +126,43 @@ def get_send_to(self, project):
             logger.debug('Tried to send notification to invalid project: %r', project)
             return []
 
+        if event:
+            owners = ProjectOwnership.get_owners(project.id, event.data)
+            if owners != ProjectOwnership.Everyone:
+                if not owners:
+                    metrics.incr(
+                        'owners.empty.match',
+                        tags={'organization': project.organization_id},
+                        skip_internal=True,
+                    )
+                    return []
+
+                from sentry.models import User
```

Comment:
This feels weird to write. But I was worried about circular import given how central this must be. any thoughts? @mattrobenolt

Reply:
It should be fine at module level.